healthScore/models.py

```diff
@@ -63,6 +63,13 @@ def create_healthcare_worker(self, email=None, password=None, **extra_fields):
         extra_fields.setdefault("is_healthcare_worker", True)
         return self._create_user(email, password, **extra_fields)
 
+    def create_superuser(self, email=None, password=None, **extra_fields):
+        extra_fields.setdefault("is_staff", True)
```

Comment:
**is_staff** key is for the healthcare admins. Should it be set to **False** while creating a superuser?

Reply:
Yeah, I initially had it set to false but after creating the superuser account and trying to get into the Django admin page it was telling me that I couldn't login to the admin page unless the super user was also flagged as is_staff. Not sure why that was

Reply:
Can flip it back to false if you think that won't matter

Reply:
Yeah, even I was facing this issue while using django admin. Things are working smoothly now since we are not using django admin. I had pushed this fix in my PR